enterprise/backup/src/test/java/org/neo4j/commandline/admin/AugmentedBlockerLocator.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2002-2018 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.commandline.admin;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+class AugmentedBlockerLocator
+{
+    static BlockerLocator fromList( AdminCommand.Blocker... blockers )
+    {
+        return commandName -> Arrays.stream( blockers ).filter( blocker -> blocker.commands().contains( commandName ) ).collect( Collectors.toList() );
+    }
```

Comment:
This is only used in one place, where we don't even give it any arguments. I don't think we need this class. The test can just pass `ArrayList::new` or something like that.